nixos/modules/services/networking/syncthing.nix

```diff
@@ -100,6 +98,17 @@ in
         '';
       };
 
+      openDefaultPorts = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Open the default ports in the firewall:
+            - TCP 22000 for transfers
+            - UDP 21027 for discovery
+          You probably only want to disable this if you are running a single instance on this machine.
```

Comment:
I found that sentence a bit difficult to read. How about "You probably want to enable this if you are running syncthing on multiple machines."?